core-feature-pack/src/main/resources/content/bin/jboss-cli.ps1

```diff
@@ -16,6 +16,7 @@ Function Set-Env {
   Set-Content -Path env:$key -Value $value
 }
 
+<<<<<<< HEAD
```

Comment:
ups, how did that get there :) there was some merge conflict when i was splitting the PR that had two commits that added/fixed powershell scripts and one that is now essentially this.